libfaketime/__init__.py

```diff
@@ -120,9 +120,8 @@ def end_callback(instance):
 class fake_time:
     def __init__(self, datetime_spec, only_main_thread=True, tz_offset=None):
         self.only_main_thread = only_main_thread
-        self.timezone_str = 'UTC'
-        if tz_offset is not None:
-            self.timezone_str = 'Etc/GMT{0:+}'.format(-tz_offset)
+
+        self.timezone_str = 'Etc/GMT{0:+}'.format(-(tz_offset or 0))
```

Comment:
If this indeed prevents a regression, it might be good to have a test that verifies the correctness of this behavior. But unless this is a quick win, I'd consider this out of scope for this pull request.

Reply:
Yup, that's reasonable; I'll take a look.

Reply:
I was wrong: it's UTC+/- that's problematic. UTC itself is fine. I added a test and reverted my change.